### PR TITLE
Use new blas scheme

### PR DESCRIPTION
**Pull request by jschueller**
Now the blas implementation is chosen at install with the new mecanism depending on the arch using the generic settings.

@conda-forge-admin, please rerender

Currently cminpack messes up builds using the blas scheme as it wants to link to libopenblas.so.0

### CODE REVIEW

**Comment by jschueller** [comment]
using nmake to try to work around an issue with the windows build

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,16 +17,16 @@ source:
...
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}  # Needed because CMake project is not declared as C
     - cmake
     - ninja
-  host:  # [local_blas_impl == 'openblas']
-    - openblas-devel  # [local_blas_impl == 'openblas']
+  host:
+    - libblas  # [unix]
```
Comment (tadeu) [inline recipe/meta.yaml:29]: Looks good. If there was a way to build both blas and noblas versions with the same recipe it would be even better ;)

**Comment by tadeu** [comment]
@jschueller, do you want to add yourself as maintainer here?

**Review by tadeu — APPROVED** [review]

**Comment by jschueller** [comment]
I'm good for now, please merge

**Comment by tadeu** [comment]
Done, sorry for the delay 😁

**Pull request merged**